.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Hey Romer, is this config working for you, I'm trying to make it work, but it just doesn't